tests/bsim/bluetooth/host/misc/disable/tests_scripts/disable_set_default_id.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+# Disable test: bt_enable and bt_disable are called in a loop.
+# Each iteration the default ID is updated
+simulation_id="disable_test_set_default_id"
+verbosity_level=2
+EXECUTE_TIMEOUT=10
```

Comment:
Please don't set the EXECUTE_TIMEOUT to small values, let the default value of 30s handle those (i.e. just delete this line)

Reply:
Will do!
FYI @cvinayak , when I was running this test locally it was stuck triggering `swi_lll_nrf5_isr`